src/joe_parentheticals.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
+import sys
+import re
+
+
+def parenthetical(parens):
+    """Determine if a string has open, balanced, or broken parenthesis"""
+    clean_str = re.sub(r'[^(+)]', '', parens)
+    while clean_str is not '':
+        if clean_str[0] == ')': return -1
+        if clean_str[-1] == '(': return 1
+        clean_str = clean_str.replace('()', '')
+    return 0
+
+
+if __name__ == '__main__':
+    str = sys.argv[1]
```

Comment:
Really bad idea to name variables after builtins.. Which is why its colored like that.